sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/GpuBroadcastToRowExec.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2021-2022, NVIDIA CORPORATION.
+ * Copyright (c) 2022-2023, NVIDIA CORPORATION.
```

Comment:
```suggestion
 * Copyright (c) 2021-2023, NVIDIA CORPORATION.
```

Reply:
So on this one, the original copyright year was incorrect. It should have only been 2022. So that's why this was changed to 2022-2023